Ethereum/ethereum-bayc/project.yaml

```diff
@@ -0,0 +1,41 @@
+# // Auto-generated , DO NOT EDIT
+specVersion: 1.0.0
+version: 0.0.1
+name: ethereum-subql-starter
+description: >-
+  This project can be use as a starting point for developing your new Ethereum
+  SubQuery project
+runner:
+  node:
+    name: '@subql/node-ethereum'
+    version: '>=3.0.0'
+  query:
+    name: '@subql/query'
+    version: '*'
+schema:
+  file: ./schema.graphql
+network:
+  chainId: '1'
+  endpoint:
+    - https://eth.api.onfinality.io/public
+  dictionary: https://gx.api.subquery.network/sq/subquery/eth-dictionary
+dataSources:
+  - kind: ethereum/Runtime
+    startBlock: 12287507
+    options:
+      abi: bayc
+      address: '0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D'
+    assets:
+      bayc:
+        file: ./abis/bayc.abi.json
+    mapping:
+      file: ./dist/index.js
+      handlers:
+        - kind: ethereum/LogHandler
```

Comment:
Can you add a Mint handler?

Reply:
https://github.com/subquery/ethereum-subql-starter/pull/42/commits/787557af34849d9c5b0b001f6c3ff4fbb68c1319